meta-mender-core/recipes-mender/mender-client/files/mender-growfs-data.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Grow File System on /data
+DefaultDependencies=no
+BindsTo=data.mount
+Conflicts=shutdown.target
+After=data.mount
+Before=shutdown.target
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/lib/systemd/systemd-growfs /data
```

Comment:
I don't get it, why do you need both this and the `parted` variant?

If you do need both, then I suggest renaming this to `mender-systemd-growfs-data.service` to make the difference more clear. For quite some time I could not figure out that the files had two different names.

Reply:
> I don't get it, why do you need both this and the parted variant?

`mender-grow-data.service` - resizes the partition and needs to run before `data.mount`
`mender-systemd-growfs-data.service` - resizes the filesystem image and can run after `data.mount`

So I need both, especially to support the "non-blocking" mode. Updated the name per your suggestion

Reply:
I see, thanks!